deploy/images.csv

```diff
@@ -34,3 +34,4 @@ KUBEVIRT_CONSOLE_PROXY_IMAGE,quay.io/kubevirt-ui/kubevirt-apiserver-proxy,KUBEVI
 AAQ_OPERATOR_IMAGE,quay.io/kubevirt/aaq-operator,AAQ_VERSION,2d1c49702fd187030dfa669791f25cefbc29e3103c2f5e0318af4c0ccead0fd2
 AAQ_SERVER_IMAGE,quay.io/kubevirt/aaq-server,AAQ_VERSION,827356ed0ad3857c1319b30cd026775dc8076eb9a1b2b62338f617429af36381
 AAQ_CONTROLLER_IMAGE,quay.io/kubevirt/aaq-controller,AAQ_VERSION,088da1c52a1b14edff7e9d314a46bd9c09c780b39ff82038fb90dc1022ae630d
+WASP_IMAGE,quay.io/ibezukh/wasp-operator-prototype,WASP_VERSION,6092433f8a55216f64b69e4a722e21fef4cedb48c97f2048969bcaf13a2ceb45
```

Comment:
is it possible to push that image to `quay.io/kubevirt/wasp-operator-prototype` ?
we should avoid referencing personal repos.